src/pymor/discretizers/builtin/gui/jupyter/threejs.py

```diff
@@ -283,7 +283,7 @@ async def finish_loading(self):
 
 
 def visualize_py3js(grid, U, bounding_box=([0, 0], [1, 1]), codim=2, title=None, legend=None,
-                    separate_colorbars=False, rescale_colorbars=False, columns=2, color_map=get_cmap('viridis')):
+                    separate_colorbars=False, rescale_colorbars=False, columns=2, color_map=mpl.colormaps['viridis']):
```

Comment:
I only see `matplotlib.cm` not `matplotlib.colormaps` in the [docs](https://matplotlib.org/stable/api/cm_api.html). Am I looking at the wrong space?

Reply:
`matplotlib.colormaps` is [here](https://matplotlib.org/stable/api/matplotlib_configuration_api.html#matplotlib.colormaps). It was added in `matplotlib` 3.5 (see [here](https://matplotlib.org/stable/api/prev_api_changes/api_changes_3.6.0.html#pending-deprecation-top-level-cmap-registration-and-access-functions-in-mpl-cm)). I guess it would be good to also support older versions.

Reply:
Ok. Great!